test/jdk/java/lang/String/Chars.java

```diff
@@ -45,6 +47,7 @@ public static void main(String[] args) {
             }
             testChars(cc, ccExp);
             testCharsSubrange(cc, ccExp);
+            testIntsSubrange(ccExp);
```

Comment:
Unlike the others, this one is not invoked for the non-ASCII part of the BMP and the next plane.
Intent or oversight?